cypress/e2e/ArticleViewing/opening_article.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+const { homePage } = require('../../support/selectors')
+
+describe('Make sure BBC.com loads properlly', () => {
```

Comment:
jau +3 file su tuo paciu user scenariju, tai gal visus testus butu galima sudeti i viena faila, arba pakeisti discribe captiona, nes cia rasai ir pries tai make sure it loads properly, aisku typo yra reikia vienos l.

Reply:
fixed ir fixed